docs/Developer's Guide.md

```diff
@@ -35,11 +35,20 @@ Here are the steps to setup the frontend for development:
 
    If it doesn't return a number, you will need to install NodeJS. There are two ways to do this:
 
-   1. Download and run the installer from [the NodeJS Website](https://nodejs.org/en/download/)
-   1. Use a tool that manages Node installations, such as
-      - For any OS: [Fast Node Manager](https://github.com/Schniz/fnm)
-      - For MacOS/Linux: [Node Version Manager](https://github.com/nvm-sh/nvm)
-      - For Windows: [NVM for Windows](https://github.com/coreybutler/nvm-windows)
+   1. Download and run the installer from [the NodeJS Website](https://nodejs.org/en/download/).  This requires admin (aka "root" or "sudoer") privileges, so if you are on a shared Linux workstation, use one of the options below.
+   1. Use a tool that manages Node installations, and use the "lts" version
+      (which means "long term support"). Options include:
```

Comment:
```suggestion
   1. Use a tool that manages Node installations, and use the Long Term Support (LTS) version. Options include:
```
This seems like a more common way of introducing an acronym. Is that clearer do you think?